### PR TITLE
x86_cpu_model: add new model Snowridge

### PR DESCRIPTION
**Pull request by nanliu-r**
ID: 1952307

Signed-off-by: Nana Liu <nanliu@redhat.com>

### CODE REVIEW

**Comment by nanliu-r** [comment]
@huangyum @vivianQizhu Hi, this bug's test is blocked by product bz 1750275, and I test it PASS manually. please help to review this, thanks.

File: qemu/tests/cfg/x86_cpu_model.cfg [hunk 1]
```diff
@@ -40,6 +40,10 @@
             only HostCpuVendor.amd
             flags = "vme sse2 sse fxsr mmx clflush pse36 pat cmov mca pge mtrr sep apic cx8 mce pae msr tsc pse de  popcnt cx16 lm nx syscall misalignsse sse4a abm lahf_lm fpu"
             model_pattern = "AMD Opteron 23xx \(Gen 3 Class Opteron%s\)"
+        - Snowridge:
+            only HostCpuVendor.intel
+            flags = "split_lock_detect gfni movdiri movdiri64b cldemote umip  "
```
Comment (huangyum) [inline qemu/tests/cfg/x86_cpu_model.cfg:45]: Extra space in the end?

Comment (nanliu-r) [reply qemu/tests/cfg/x86_cpu_model.cfg]: update and thanks.

**Review by huangyum — APPROVED** [review]
ACK

**Review by vivianQizhu — APPROVED** [review]

**Comment by vivianQizhu** [comment]
Thanks @nanliu-r @huangyum

**Pull request merged**